tool/net/redbean.c

```diff
@@ -3134,16 +3170,26 @@ static char *GetLuaResponse(void) {
   return p;
 }
 
+static bool IsLoopbackClient() {
+  uint32_t ip;
+  uint16_t port;
+  GetClientAddr(&ip, &port);
```

Comment:
Please change this to GetRemoteAddr() so that backtraces won't leak to external hosts in well-configured proxy scenarios.

Reply:
Good point; pushed the fix along with a couple of other changes. The error location is reported now as `"/sql.lua"` instead of `[string "/sql.lua"]`.